mrmustard/physics/triples.py

```diff
@@ -476,6 +475,75 @@ def fock_damping_Abc(n_modes: int) -> Union[Matrix, Vector, Scalar]:
     return A, b, c
 
 
+def gaussian_channel_XYd_Abc(
```

Comment:
Hey Kasper, thanks for this! We already have this functionality though ^^'
I'll show you on Monday how I intend to make these components available